### DISCUSSION TITLE
Zemismart curtain motor BCM500DS-TYZ-B

### DISCUSSION THREAD

**Discussion by Tomas778** [Koenkk/zigbee2mqtt#13461] [Devices]
Hi everyone,
does anyone have tried to pair Zemismart curtain motor BCM500DS-TYZ-B with Zigbee2Mqtt. 
In manual it shows: On 1-button units; press the set key 3 times in 5 seconds until the LED flashes blue. In reality It keeps flashing red (doesn't seems that it have blue LED), but It doesn't show up in Zigbee2Mqtt integration, just wondering what is wrong and maybe someone could help.

**Comment by umkaalmagor** [comment]
three short quick presses on the button. the led will flash red
